cbr-zone-module/README.md

```diff
@@ -0,0 +1,3 @@
+# Default example
+
+Example to create a zone for Context Based Restrictions
```

Comment:
this is not an example. This is an actual (sub)module (called by the root level module), so this readme should be titled `CBR Zone Module` and should describe what the module does